File: HelloWorld-all/HelloWorld.cpp
#include <iostream>               //std::cout 要用到的头文件  
#include <stdio.h>                //标准输入输出头文件  
using namespace std;

int main()  
{  
  printf("Hello,World!--Way 1\n");    //printf 语句打印,C语言格式的
  puts("Hello,World!--Way 2");        //puts 语句  
  puts("Hello," " " "World!--Way 3"); //字符串拼接  
  cout << "Hello,World!--Way 4" << endl; //C++ 教科书上写法  
  return 0;
}

File: HelloWorld-all/HelloWorld.rb
#可用 print 语句打印  
print "Hello,World!\n"   
#可用 puts 语句打印  
puts  "Hello,World!\n"   
#可以先声明一个变量，然后再用 puts 语句  
a = "Hello,World!\n"     
puts a  
#可以先写个函数再调用  
def say(name)  
   "Hello,#{name}"  
end  
puts say("World!")
